src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.hooks.ts

```diff
@@ -124,14 +91,10 @@ export const useFeedbackNotes = ({ entity: { id, isFlow } }: FeedbackNotesProps)
     const commonParams = { appletId, noteId };
 
     if (isFlow && submitId) {
-      deleteFlowNote({ ...commonParams, submitId, flowId: id });
-
-      return;
+      deleteNote({ ...commonParams, submitId, flowId: id });
```

Comment:
let's move the condition logic `if (isFlow && submitId) {...}` as a part of `deleteNote` function